src/lib/factories/sampleFactory.ts

```diff
@@ -27,11 +27,11 @@ export const sampleFactory = Factory.define<Sample>(
 );
 
 export const tissueFactory: Factory<Tissue> = Factory.define<Tissue>(
-  ({ sequence, params, associations }) => ({
```

Comment:
What does it mean that these `sequence` parameters have been removed?

Reply:
@khelwood sequence parameter is not used in sampleFactory and in other places , so there were ts compile warnings about "Unused parameter sequence"